package.json

```diff
@@ -21,7 +21,9 @@
     "zarr.umd.js",
     "zarr.umd.js.map"
   ],
-  "sideEffects": false,
+  "sideEffects": [
+    "./zarr.mjs"
```

Comment:
This ensures that the codecs aren't tree-shaken with webpack. Imports from `zarr/core` are tree-shaken.